demisto_sdk/commands/content_graph/strict_objects/layout_rule.py

```diff
@@ -0,0 +1,48 @@
+from typing import List, Optional, Union
+
+from pydantic import Field
+
+from demisto_sdk.commands.content_graph.strict_objects.common import (
+    DESCRIPTION_DYNAMIC_MODEL,
+    BaseStrictModel,
+    create_model,
+)
+
+
+class FilterSchema(BaseStrictModel):
+    SEARCH_FIELD: str
+    SEARCH_TYPE: str
+    SEARCH_VALUE: str
+
+
+class AndSchema(BaseStrictModel):
+    AND: Optional[List[Union["FilterSchema", "OrSchema", "AndSchema"]]] = None
```

Comment:
`Union["FilterSchema", "OrSchema", "AndSchema"]` can be a variable, then reused between `OR` and `AND`